GOVERNANCE.md

```diff
@@ -0,0 +1,53 @@
+
+Inspired by [ANSI’s Essential Requirements for Due Process](https://share.ansi.org/Shared%20Documents/Standards%20Activities/American%20National%20Standards/Procedures,%20Guides,%20and%20Forms/2020_ANSI_Essential_Requirements.pdf), the Working Group adheres to consensus-based due process requirements.  These requirements apply to activities related to the development of consensus for approval, revision, reaffirmation, and withdrawal of specifications.  Due process means that any person (organization, company, government agency, individual, etc.) with a direct and material interest has a right to participate by: a) expressing a position and its basis, b) having that position considered, and c) having the right to appeal. Due process allows for equity and fair play. The following constitute the minimum acceptable due process requirements for the development of consensus.
+
+**3.1.	Openness.**  Participation shall be open to all persons who are directly and materially affected by the activity in question. There shall be no undue financial barriers to participation. Voting membership on the consensus body shall not be conditional upon membership in any organization, nor unreasonably restricted on the basis of technical qualifications or other such requirements.  Membership in a Working Group’s parent organization, if any, may be required.
+
+**3.2.	Lack of Dominance.**  The development process shall not be dominated by any single interest category, individual or organization. Dominance means a position or exercise of dominant authority, leadership, or influence by reason of superior leverage, strength, or representation to the exclusion of fair and equitable consideration of other viewpoints.
+
+**3.3.	Balance.**  The development process should have a balance of interests. Participants from diverse interest categories shall be sought with the objective of achieving balance.
+
+**3.4.	Coordination and Harmonization.**  Good faith efforts shall be made to resolve potential conflicts between and among deliverables developed under this Working Group and existing industry standards.
+
+**3.5.	Consideration of Views and Objections.**  Prompt consideration shall be given to the written views and objections of all Participants.
+
+**3.6.	Written procedures.**  This governance document and other materials documenting the specification development process shall be available to any interested person.
+
+## 4.	Specification Development Process.  
+
+**4.1.	Pre-Draft.**  Any Participant may submit a proposed initial draft document as a candidate Draft Specification of the Working Group.  The Maintainer(s) will designate each submission as a “Pre-Draft” document.
+
+**4.2.	Draft.**  Each Pre-Draft document of the Working Group must first be Approved to become a “Draft Specification”.  Once the Working Group approves a document as a Draft Specification, the Draft Specification becomes the basis for all going forward work on that specification.

```

Comment:
Correct.

Reply:
Languages changes @RexJaeschke recommends also need to be applied here
#482 
#483

Reply:
Re the use of "S/specification," our current thinking is to use "Standard" and that "Specification" (uppercase) *not* be used. Kris has come up with new section names that eliminate such uses of. That said, we *could* revert to "Specification" instead, but should think that through a bit. Then there is the question of whether we should ever use "specification" (lowercase) anywhere regardless of the previous sentence outcome. The original confusion for me arose from the use of "Specification" in section headers, which suggested they were part of the requirements while other sections without that were not, simply because of the implicit understanding of what "Specification" means.

Reply:
@copiesofcopies I think I was still slightly off (I realised after reading @scooter99boston's clarification). Rather, our process is:
- various contributions are made by participants, 
  - consensus (of the Standard Working Group) is checked (by the maintainers) on inclusion, 
  - PRs are reviewed by an editor and maintainers then merged to our master branch 
- Maintainers designated the master branch as a 'Pre-Draft' version of the standard 
- That is then submitted back to the Standards working group for review and vote on adoption - which progresses it to the status of a Draft (4.3).
- We then do any work necessary to publish the standard, at which point it is published (4.4)

While writing this I spotted a further issue:
> **4.3.	Working Group Approval.**  Once the Working Group believes it has achieved the objectives for its specification **as described in the Scope,**

I don't know what "the Scope" is or where it's defined as that document (from the [Standards Project Blueprint](https://github.com/finos/standards-project-blueprint/blob/master/2._Scope.md)) is not part of this PR. The scope is defined in the FDC3 Charter here: https://fdc3.finos.org/docs/fdc3-charter

Perhaps we should pull that content in to create the scope document OR refer out to it. If we're trying to head towards the full CSL I would suggest the former is the way to go. Do you agree?

Reply:
@copiesofcopies as its referred to, should we bring in the scope doc (from the [Standards Project Blueprint](https://github.com/finos/standards-project-blueprint/blob/master/2._Scope.md)) and populate with details from https://fdc3.finos.org/docs/fdc3-charter ?

Can you do that or should I raise a PR (into yours?) to do so?

Reply:
Hi @kriswest thanks for catching this. I think it makes sense to include a scope file as you suggested. I've added a [proposed scope](https://github.com/finos/FDC3/pull/485/files#diff-232847d7bce0ee80595653fc1167fa9fbd84a1eef4659c4a93cb7dddf8d59787) to the pull request, incorporating the FDC3 mission (with minor changes to adapt it to the purpose) and scope (unchanged).

Reply:
Awesome, thanks @copiesofcopies.

I've raised a separate issue, #529, to cover adding a description of how we implement this governance to the README and website - hence I think this is now good to go!